tests/Makefile

```diff
@@ -0,0 +1,36 @@
+
+test-setup: setup-modules setup-config
+
+# Install a test site.
+install-site:
+	drush site-install ding2
+
+# Setup test modules on site.
+setup-modules:
+	# We use the connie test provider for testing. Enable it and
+	# disable the default alma.
+	drush en -y connie ting_covers_placeholder
+	drush dis -y alma ting_covers_addi
+
+
+	# Enable mocking openlist provider.
+	drush en -y connie_openlist
+
+setup-config:
+	# Configure ting.module
+	drush vset -y ting_agency 100200
+	drush vset -y ting_search_url 'http://oss-services.dbc.dk/opensearch/4.2/'
```

Comment:
No longer a valid url.
We are at https://oss-services.dbc.dk/opensearch/5.0/ now